extensions/ql-vscode/test/vscode-tests/cli.ts

```diff
@@ -42,3 +42,11 @@ export function itWithCodeQL() {
 
   return it;
 }
+
+export function skipIfTrue(condition: () => Thenable<boolean>) {
+  return beforeEach(async () => {
+    if (await condition) {
```

Comment:
Should be
```suggestion
    if (await condition()) {
```
as otherwise this will always be true, assuming a function object is truthy.

Or convert `condition` back into a `Thenable<boolean>`. I don't think the extra layer of being function is necessary to make this work.

As an extra side point, what benefit does using `Thenable` give over using `Promise`? I'm curious to know which one we should use in general.

Reply:
Actually, I don't think the function is really doing much. I'm moving to vanilla `beforeEach...pending`.